.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY = HEVGp9F
```

Comment:
Smaht

Reply:
Yeah make sure you put this in the .gitignore file! Don't want to merge this for others to see.